src/core/settings.c

```diff
@@ -602,6 +608,11 @@ QuicSettingApply(
         Destination->IsSet.GreaseQuicBitEnabled = TRUE;
     }
 
+    if (Source->IsSet.ReliableResetEnabled && (!Destination->IsSet.ReliableResetEnabled || OverWrite)) {
```

Comment:
Move this down, after `EncryptionOffloadAllowed`.

Reply:
Done